### PR TITLE
chore(fluent-accordion): Add screenshot tests

### PR DESCRIPTION
**Pull request by miroslavstastny**
Adds VR tests for `fluent-accordion` WC.
Adds Story decorator which sets light/dark theme and LTR/RTL direction for the story.

### CODE REVIEW

File: apps/vr-tests-web-components/.storybook/preview.mjs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { WCThemeDecorator } from '../src/utilities/WCThemeDecorator'
```
Comment (fabricteam) [inline apps/vr-tests-web-components/.storybook/preview.mjs:1]: ## [🕵 FluentUI-web-components Open the Visual Regressions report to inspect the 26 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28441&commitId=a13f59c53bd5c8bb39089790e4ff8edc5cca8d36&env=prod)

✅ There was 22 screenshots added, 2 screenshots removed, 0 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 26 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Accordion With Custom Icons.normal.chromium.png|1447| Changed|
|Accordion.Accordion With Custom Icons.opened.chromium.png|1858| Changed|
|Accordion.Accordion With Custom Icons - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons - Dark Mode.opened.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons - RTL.normal.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons - RTL.opened.chromium.png|0| Added|
|Accordion.Disabled - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Disabled - Dark Mode.opened.chromium.png|0| Added|
|Accordion.Disabled - RTL.normal.chromium.png|0| Added|
|Accordion.Disabled - RTL.opened.chromium.png|0| Added|
|Accordion.Disabled.normal.chromium.png|0| Added|
|Accordion.Disabled.opened.chromium.png|0| Added|
|Accordion.Expand Icon Position End - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End - Dark Mode.opened.chromium.png|0| Added|
|Accordion.Expand Icon Position End - RTL.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End - RTL.opened.chromium.png|0| Added|
|Accordion.Expand Icon Position End.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End.opened.chromium.png|0| Added|
|Accordion.Size - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Size - Dark Mode.opened.chromium.png|0| Added|
|Accordion.Size - RTL.normal.chromium.png|0| Added|
|Accordion.Size - RTL.opened.chromium.png|0| Added|
|Accordion.Size.normal.chromium.png|0| Added|
|Accordion.Size.opened.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons RTL.normal.chromium.png|0| Removed|
|Accordion.Accordion With Custom Icons RTL.opened.chromium.png|0| Removed|

</details>


<div id="vrtCommentFluentUI-web-components"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/07f3fe5fc625defc06f6ec5376a4ec96c2816e01'>07f3fe5fc625defc06f6ec5376a4ec96c2816e01</a></sub>
<div id="bundleSizeComment9423"/>

File: apps/vr-tests-web-components/.storybook/preview.mjs (hunk 1 above)
Comment (miroslavstastny) [reply apps/vr-tests-web-components/.storybook/preview.mjs:1]: #28451 
#28452

File: apps/vr-tests-web-components/src/stories/accordion/accordion.stories.tsx [hunk 2]
```diff
@@ -27,22 +22,64 @@ export default {
   ],
 };
 
-const add20Filled = `<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+const add20Filled = `<svg width="20" height="20" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" style="fill: currentColor;">
```
Comment (levithomason) [inline apps/vr-tests-web-components/src/stories/accordion/accordion.stories.tsx:25]: This should probably be reverted to match the storybook story?

File: apps/vr-tests-web-components/src/stories/accordion/accordion.stories.tsx [hunk 3]
```diff
@@ -27,22 +22,64 @@ export default {
...
+
 const subtract20Filled = `<svg
   width="20"
   height="20"
   viewBox="0 0 20 20"
-  fill="none"
   xmlns="http://www.w3.org/2000/svg"
+  style="fill: currentColor;"
 >
-  <rect x="3" y="9.25" width="14" height="1.5" rx="0.75" fill="#212121" />
```
Comment (levithomason) [inline apps/vr-tests-web-components/src/stories/accordion/accordion.stories.tsx:43]: Here as well.

File: apps/vr-tests-web-components/src/utilities/WCThemeDecorator.tsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
```
Comment (levithomason) [inline apps/vr-tests-web-components/src/utilities/WCThemeDecorator.tsx:1]: Note, file extension is tsx.

**Review by levithomason — APPROVED** [review]

File: apps/vr-tests-web-components/.storybook/preview.mjs (hunk 1 above)
Comment (fabricteam) [inline apps/vr-tests-web-components/.storybook/preview.mjs:1]: ## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28441&commitId=e76b5076697920c88d1bca77b5ca511889a3a48e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Accordion</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Accordion With Custom Icons.normal.chromium.png|1447| Changed|
|Accordion.Accordion With Custom Icons - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons - RTL.normal.chromium.png|0| Added|
|Accordion.Disabled - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Disabled - RTL.normal.chromium.png|0| Added|
|Accordion.Disabled.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End - RTL.normal.chromium.png|0| Added|
|Accordion.Expand Icon Position End.normal.chromium.png|0| Added|
|Accordion.Size - Dark Mode.normal.chromium.png|0| Added|
|Accordion.Size - RTL.normal.chromium.png|0| Added|
|Accordion.Size.normal.chromium.png|0| Added|
|Accordion.Accordion With Custom Icons RTL.normal.chromium.png|0| Removed|
|Accordion.Accordion With Custom Icons RTL.opened.chromium.png|0| Removed|
|Accordion.Accordion With Custom Icons.opened.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Pull request merged**